script.js

```diff
@@ -0,0 +1,841 @@
+const FUNCTIONAL_KEYS = {
+  shift: false,
+  shiftRight: false,
+  ctrl: false,
+  ctrlRight: false,
+  alt: false,
+  altRight: false,
+  windows: false,
+  caps: false,
+};
+
+if (!window.localStorage.getItem("virtualKeyBoardLang")) window.localStorage.setItem("virtualKeyBoardLang", "en");
+
+const vKeyBoard = {

```

Comment:
Isn't it better to create a class, that accept textarea as input, and has methods to focus and insert symbols?

Reply:
Created class which facilitates usage of texarea

https://github.com/BlackTruth/virtual-keyboard/blob/6297a0817c03affb1948c9e73a6352687e6ff449/src/classes/VirtualKeyboardInput.js#L1-L5